File: 27.Remove Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int i(0), j(0);
        for(i=0;i<nums.size();i++){
            if(nums[i]!=val){
                nums[j]=nums[i];
                j++;
            }
        }
        return j;
    }
};

File: 27.Remove Element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        i = 0
        for j, x in enumerate(nums):
            if x != val:
                nums[i] = nums[j]
                i = i + 1
        return i
                
